src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -2079,14 +2088,15 @@
         },
         "PORT_QOS_MAP": {
            "Ethernet0": {
-              "dot1p_to_tc_map" : "Dot1p_to_tc_map1",
-              "dscp_to_tc_map": "Dscp_to_tc_map1",
-	          "tc_to_queue_map": "tc_to_q_map1",
-	          "tc_to_pg_map": "tc_to_pg_map1",
-	          "pfc_to_queue_map": "pfc_prio_to_q_map1",
-	          "pfc_to_pg_map"   : "pfc_prio_to_pg_map1",
-	          "pfc_enable"      : "3,4",
-	          "pfcwd_sw_enable" : "3,4"
+               "dot1p_to_tc_map" : "Dot1p_to_tc_map1",
+               "dscp_to_tc_map": "Dscp_to_tc_map1",
+               "tc_to_queue_map": "tc_to_q_map1",
+               "tc_to_pg_map": "tc_to_pg_map1",
+               "pfc_to_queue_map": "pfc_prio_to_q_map1",
+               "pfc_to_pg_map"   : "pfc_prio_to_pg_map1",
+               "pfc_enable"      : "3,4",
+               "pfcwd_sw_enable" : "3,4",
+               "scheduler"       : "port_scheduler"
             },
             "Ethernet4": {
```

Comment:
any reason Ethernet0 and Ethernet4 does not have the same number of parameters as well as in one we have indentation and in the other one we dont.

Reply:
@liat-grozovik 
Usually, we do not have the same port configuration on different ports because we want to have different configurations on different ports to cover different scenarios.
Regarding the indentation, I fixed the indentation in port `Ethernet0` because I modified it in my PR. IMO there are many indentation issues in this file, which should not be fixed in this PR because I'm focusing on missing `scheduler`  in this PR.

Reply:
ACK. @qiluo-msft FYI you might want to take it to the yang subgroup as a reference and to check if need to fix the file(s)